lib/mountain_view/engine.rb

```diff
@@ -1,5 +1,6 @@
-require 'mountain_view'
 require 'rails'
+require 'mountain_view'
+require 'mountain_view/component'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.